src/clusterfuzz/_internal/system/environment.py

```diff
@@ -24,6 +24,7 @@
 import yaml
 
 from clusterfuzz._internal import fuzzing
+from clusterfuzz._internal.platforms import android
```

Comment:
Hmm... so I meant to import at the top `if is_android`.
@marktefftech Do you know why we gated this import in the first place?

Reply:
Unfortunately no, the imports have been this way since Oliver committed the original Android implementation in 2019